### PR TITLE
Valeria contentful integration

### PR DESCRIPTION
**Pull request by valeriafurin**
Hi there, I integrated Contentful and Apollo and did a proper App set-up.
Please review, thank you!

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+export REACT_APP_CONTENTFUL_TOKEN=4J0wH_kob0-C2-7MwgL5JqNJkTLtjDql0TrSJ9gFdCA
```
Comment (barsukov) [inline .env:1]: Please add .env to .gitignore and since and regenerate the token ;)

File: package.json [hunk 2]
```diff
@@ -3,12 +3,19 @@
...
   "private": true,
   "dependencies": {
+    "@apollo/client": "^3.3.11",
+    "@apollo/react-hooks": "^4.0.0",
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "@types/jest": "^26.0.20",
+    "@types/node": "^14.14.31",
+    "apollo-boost": "^0.4.9",
```
Comment (barsukov) [inline package.json:13]: you can delete this package

File: src/App.tsx [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { gql, useQuery } from "@apollo/client";
+import "./App.css";
+
+const query = gql`
```
Comment (barsukov) [inline src/App.tsx:4]: Please extract this query to own file and reference here

**Review by barsukov — CHANGES_REQUESTED** [review]
for the first integration quite good, please delete yarn.lock, and adjust a few comments 👍

File: .env (hunk 1 above)
Comment (valeriafurin) [reply .env:1]: Already did that :)

File: src/query.tsx [hunk 4]
```diff
@@ -0,0 +1,14 @@
+import { gql } from "@apollo/client";
+
+export const query = gql`
```
Comment (barsukov) [inline src/query.tsx:3]: please name this query regarding the name of the requested data and do not forget to rename the file as well.

**Review by barsukov — CHANGES_REQUESTED** [review]

File: src/query.tsx (hunk 4 above)
Comment (valeriafurin) [reply src/query.tsx]: Done :)

**Review by barsukov — APPROVED** [review]
👍

**Pull request merged**